### PR TITLE
first assigments

### PR DESCRIPTION
**Pull request by vlitan**
first 3 assigments done. 
(log(n) fibbonaci missing)

### CODE REVIEW

File: Students/Litan A. Virgil/assig13.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+class assig3
+{
+	public static void main (String[] args) throws java.lang.Exception
+	{
+		System.out.println("3 digit : " + getPali(2));
+		System.out.println("3 digit : " + getPali(3));
+		System.out.println("4 digit : " + getPali(4));
+	}
+
+	private static long getPali(int digits)
```
Comment (AlexBondor) [inline Students/Litan A. Virgil/assig13.java]: Don't be lazy and write the full name here:D

File: Students/Litan A. Virgil/assig11.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+import java.util.Scanner;
+
+
+class assig1
+{
```
Comment (AlexBondor) [inline Students/Litan A. Virgil/assig11.java]: Should be fine if you stick to this type of braces unless you won't be writing Java code inside a company. If you intend to do that at some point then go for the other one.

File: Students/Litan A. Virgil/assig11.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	{
+		int input;
+		long result;
+
+		input = getInput();
+		result = getResult(input);
+		System.out.println(result);
+		printSecondTweak();
+	}
+	private static int getInput()
```
Comment (AlexBondor) [inline Students/Litan A. Virgil/assig11.java]: OOP style says we should use static as less as possible. As I told you in the class.. only for **helper** classes. Avoid using it as much as possible. It shouldn't have been the case to use it here.

File: Students/Litan A. Virgil/assig11.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+import java.util.Scanner;
+
+
+class assig1
```
Comment (AlexBondor) [inline Students/Litan A. Virgil/assig11.java]: Capital letter for class names

File: Students/Litan A. Virgil/assig11.java (hunk 3 above)
Comment (vlitan) [inline Students/Litan A. Virgil/assig11.java]: firstly I used did not used ¨static¨, but I got an error like ¨non static method called in static method¨ while trying to use it in main. So I did it this way...

File: Students/Litan A. Virgil/assig11.java (hunk 2 above)
Comment (vlitan) [inline Students/Litan A. Virgil/assig11.java]: C habbits, sorry

**Pull request merged**